Create MindShift-ToHandleAnxiety-App.md

Rami24t opened the pull request:
Mindshift is a Tool for relaxation skills.

HelonDCI reviewed (changes requested):
Proposed changes need to follow the format listed in `CONTRIBUTING.md`

Rami24t commented:
I fixed the problem. I removed the extra file and edited the readme file in accordance with the required format.
Added One section and one app to the readme file.

HelonDCI commented on README.md:
```diff
@@ -32,5 +33,8 @@ See the [contributing guide](CONTRIBUTING.md) for detailed instructions on how t
 
 ### Knowledge Management
 
+### Mind Apps
```
I don't think this is a sub category, I think this app is a knowledge management app so just remove this.

HelonDCI commented on README.md:
```diff
@@ -32,5 +33,8 @@ See the [contributing guide](CONTRIBUTING.md) for detailed instructions on how t
 
+### Mind Apps
+
+- [MindShift](https://www.anxietycanada.com/resources/mindshift-cbt/) - Is a great App to learn relaxation skills, help develop new ways of thinking, and suggest healthy activities. It can help you reframe your anxious thoughts and make lasting behavioral changes. It has lots of useful tools, including a thought journal, chill zone with guided meditations, and a “quick relief” tool if you’re feel ...
```
I think we can shorten the description here to just the first sentence.

HelonDCI reviewed (changes requested):
See comments for requested changes.

Rami24t commented:
Ok . All done.
